src/routes/solid-router/reference/data-apis/create-async.mdx

```diff
@@ -2,21 +2,41 @@
 title: createAsync
 ---
 
-<Callout>
-	This is light wrapper over [`createResource`](/reference/basic-reactivity/create-resource)
-	which serves as a stand-in for a future primitive being brought to Solid core
-	in 2.0.
-</Callout>
-
 An asynchronous primitive with a function that tracks similar to `createMemo`.
 `createAsync` expects a promise back that is then turned into a Signal.
 Reading it before it is ready causes Suspense/Transitions to trigger.
 
-```jsx
-const user = createAsync(() => getUser(params.id));
-```
-
-<Callout>
+<Callout type="caution">
 	Using `cache` in `createResource` directly will not work since the fetcher is
 	not reactive. This means that it will not invalidate properly.
 </Callout>
+
+This is light wrapper over [`createResource`](/reference/basic-reactivity/create-resource) which serves as a stand-in for a future primitive being brought to Solid core in 2.0. It is recommended that `createAsync` be used in favor of `createResource` specially when in a **SolidStart** app because `createAsync` works better in conjunction with the [cache](/solid-router/reference/data-apis/cache) helper.
+
+
+
+```tsx title="component.tsx" {6,10}
+import { createAsync } from "solid/router";
+import { Suspense } from "solid-js";
+import { getUser } from "./api";
+
+export function Component () => {
+	const user = createAsync(() => getUser(params.id));
+
+	return (
+		<Suspense fallback="loading user...">
+			<p>{user()}</p>
+		</Suspense>
+	);
+```
```

Comment:
I believe this is breaking the markdown. At the very least, I'm seeing the following text render as code in the MD vs text

Reply:
update: we checked together and it's fine now!